CourseTemplate/CourseTemplate.Data/CourseTemplateDbContext.cs

```diff
@@ -4,17 +4,12 @@ namespace CourseTemplate.Data;
 
 public class CourseTemplateDbContext : DbContext
 {
-	public DbSet<CourseTemplate> CourseTemplates { get; set; }
-	public CourseTemplateDbContext()
```

Comment:
How this impact on migrations applying?

Reply:
This should be deleted in order to launch correctly application in docker. All configuration are done in startup